packages/blade/src/components/Carousel/Indicators/Indicators.tsx

```diff
@@ -14,8 +14,7 @@ const Indicators = (props: IndicatorsProps): React.ReactElement => {
       {...makeAccessible({ role: 'tablist', label: 'Slides' })}
     >
       {new Array(props.totalItems).fill(0).map((_, idx) => {
-        const isResponsive = visibleItems === 'autofit';
-        let _visibleItems = visibleItems as 1 | 2 | 3;
+        let _visibleItems = visibleItems!;
```

Comment:
didn't quite understand why are are now getting isresponsive from context? can you elaborate?

Reply:
Previously the isResponsive was getting derived from `visibleItems === 'autofit'` which was causing issues because this check will always fail on mobile since on the main Carousel.web file we check for `isMobile` and set visibleItems to `1`.

To fix this I separated isResponsive & visibleItems to different values to reduce ambiguity.